spec/lib/move_params_spec.rb

```diff
@@ -7,50 +7,6 @@
 
   it { expect(subject.to_s).to eql 'GrapeSwagger::DocMethods::MoveParams' }
 
-  describe 'find_post_put' do
-    let(:paths) { {} }
-
-    describe 'paths empty' do
-      specify { expect { |b| subject.find_post_put(paths, &b) }.not_to yield_control }
-    end
-
-    describe 'no post/put given' do
-      let(:paths) do
-        {
-          :'/foo' => { get: {}, delete: {} },
-          :'/bar/{key}' => { get: {}, delete: {} }
-        }
-      end
-      specify { expect { |b| subject.find_post_put(paths, &b) }.not_to yield_control }
-    end
-
-    describe 'no post/put given' do
-      let(:paths) do
-        {
-          :'/foo' => { get: {}, delete: {}, post: {}, put: {}, patch: {} },
-          :'/bar/{key}' => { get: {}, delete: {}, post: {}, put: {}, patch: {} }
-        }
-      end
-      let(:expected) do
-        [
-          { post: {}, put: {}, patch: {} },
-          { post: {}, put: {}, patch: {} }
-        ]
-      end
-      specify { expect { |b| subject.find_post_put(paths, &b) }.to yield_control.twice }
-      specify { expect { |b| subject.find_post_put(paths, &b) }.to yield_successive_args *expected }
-    end
-  end
-
-  describe 'find_definition_and_params' do
-    specify do
-      subject.instance_variable_set(:@definitions, definitions)
-      subject.find_definition_and_params(found_path[:post], :post)
-      expect(definitions.keys).to include 'InBody'
-      expect(definitions['postRequestInBody'].keys).to_not include :description
```

Comment:
in principal yeap … cause now not the whole set of paths and definitions are the parameters of the method → see here https://github.com/ruby-grape/grape-swagger/pull/467/files#diff-7af39528837333c960830618fb6a52e9R162

maybe I should rename it